PWGHF/Tasks/taskMcEfficiency.cxx

```diff
@@ -790,9 +836,19 @@ struct HfTaskMcEfficiency {
                      aod::McCollisionLabels const& colls)
   {
     std::vector<int> pdgCodes{Pdg::kDS, Pdg::kLambdaCPlus};
-    candidate3ProngMcLoop<false, true, true>(candidates, tracks, mcParticles, colls, pdgCodes);
+    candidate3ProngMcLoop<false, true, true, false>(candidates, tracks, mcParticles, colls, pdgCodes);
   }
   PROCESS_SWITCH(HfTaskMcEfficiency, processMcDsLc, "Process MC for Ds+ and Lc signals", false);
+
+  void processMcLcXic(soa::Join<aod::HfCand3Prong, aod::HfSelLc, aod::HfSelXicToPKPi> const& candidates,
+                      TracksWithSelectionMC const& tracks,
+                      aod::McParticles const& mcParticles,
+                      aod::McCollisionLabels const& colls)
+  {
+    std::vector<int> pdgCodes{Pdg::kLambdaCPlus, Pdg::kXiCPlus};
+    candidate3ProngMcLoop<false, false, true, true>(candidates, tracks, mcParticles, colls, pdgCodes);
+  }
+  PROCESS_SWITCH(HfTaskMcEfficiency, processMcLcXic, "Process MC for Lc and XicPlus signals ", false);
```

Comment:
I would avoid to add these combinations, which easily grow uncontrollably (e.g. why not DsXic, DplusXic, etc.)